Automattic/jetpack
# Jetpack mu and wpcomsh - hide wpcom items for non wpcom site users, hide hosting card from non-admins (re-ship)

Fixes # https://github.com/Automattic/dotcom-forge/issues/7892

## Proposed changes:

* Reshipping https://github.com/Automattic/jetpack/pull/38155 to hide wpcom specific wpadmin items when user is not connected with wpcom account. See the [linked PR](https://github.com/Automattic/jetpack/pull/38155) for more info.
* This was reverted last week due to me being overly cautious. Towards end of day before going afk for an extended weekend, there were unanswered questions and undiagnosed bugs. The 'bug' was identified as being caused elsewhere and fixed separately.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Follow testing instructions on https://github.com/Automattic/jetpack/pull/38155 

In addition:
* Set the site back to 'default' view.
* Visit calypso and verify that "Plugins -> Scheduled Updates" in the calypso left nav is visible, and that these added checks does not remove that in that context.
